server: docs - refresh and tease a little bit more the http server (#5718)
ggml-org/llama.cpp: README.md

- [x] [MobileVLM 1.7B/3B models](https://huggingface.co/models?search=mobileVLM)
- [x] [Yi-VL](https://huggingface.co/models?search=Yi-VL)

> **HTTP server**
> 
> We offer a fast, lightweight [OpenAI API](https://github.com/openai/openai-openapi) compatible HTTP server.
> 
> [LLaMA.cpp web server](./examples/server) can be used to serve local models and easily connect them to existing clients.

Reviewer: ```suggestion
**HTTP server**

[llama.cpp web server](./examples/server) is a lightweight [OpenAI API](https://github.com/openai/openai-openapi) compatible HTTP server that can be used to serve local models and easily connect them to existing clients.
```
